src/DoctrineMongoODMModule/Module.php

```diff
@@ -42,8 +41,7 @@
  * @since   0.1.0
  * @author  Tim Roediger <superdweebie@gmail.com>
  */
-class Module implements
-    BootstrapListenerInterface,
+final class Module implements
```

Comment:
Can you rollback to `class`? We don't know how people are using `Module` right know. I don't think that someone is extending this class but we can not know